src/github.com/matrix-org/dendrite/tools/create-room-events/main.go

```diff
@@ -0,0 +1,128 @@
+// Generate a list of matrix room events for load testing.
+// Writes the events to stdout by default.
+package main
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"github.com/matrix-org/dendrite/roomserver/api"
+	"github.com/matrix-org/gomatrixserverlib"
+	"golang.org/x/crypto/ed25519"
+	"os"
+	"strconv"
+	"time"
+)
+
+var (
+	// The name of the matrix server to generate events for.
+	serverName = defaulting(os.Getenv("SERVER_NAME"), "localhost")
+	// The ID of the key used to sign the events.
+	keyID = defaulting(os.Getenv("KEY_ID"), "ed25519:auto")
+	// Base64 encoded private key to sign the events with.
+	privateKeyString = defaulting(os.Getenv("PRIVATE_KEY"), defaultKey)
+	// The room ID to generate events in.
+	roomID = defaulting(os.Getenv("ROOM_ID"), "!roomid:"+serverName)
+	// The user ID to use for as the event sender.
+	userID = defaulting(os.Getenv("USER_ID"), "@userid:"+serverName)
+	// The number of m.room.messsage events to generate.
+	messageCount = defaulting(os.Getenv("MESSAGE_COUNT"), "10")
+	// The output format to use for the messages.
+	//	INPUT -> api.InputRoomEvent
+	//  RAW   -> gomatrixserverlib.Event
```

Comment:
Adding more terminology for a dendrite-only tool feels sadness making to me. Can we not just use `Event` and `InputRoomEvent` (the actual type names)?

Also, tabs!